examples/nomad.yaml

```diff
@@ -9,20 +9,20 @@
 
 # This example requires Lima v0.7.0 or later.
 
-# Image is set to focal (20.04 LTS) for long-term stability
+# Image is set to jammy (22.04 LTS) for long-term stability
```

Comment:
TODO: Revert this until https://github.com/Roblox/nomad-driver-containerd/pull/133 gets released